docs/source/dataframe-api.rst

```diff
@@ -260,6 +257,127 @@ Series
    Series.where
 
 
+Accessors
+~~~~~~~~~
+Similar to pandas, Dask provides dtype-specific methods under various accessors.
+These are separate namespaces within :class:`Series` that only apply to specific data types.
+
+Datetime Accessor
+*****************
+
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor_attribute.rst
```

Comment:
you might have to split this in two parts: one for properties like `Series.dt.day` (using the `accessor_attribute` template), and another for the methods (using the `accessor_method` template).

Reply:
Ah, nice catch. That's a bit unfortunate that this is needed. Knowing nothing about how sphinx works - is this a sphinx limitation, or is it something that this extension could handle with the same template for both?

Reply:
it's more like that is what `pandas` did when I last looked. It would probably possible to detect whether the attribute is a descriptor or a method and automatically use the appropriate documenter, but I never really tried.

Also, at least for the projects I work on the split serves as a reminder that those are called differently (I'm not too familiar with `dask.dataframe`, though)